docs/SDCard_Updates.md

```diff
@@ -41,7 +41,7 @@ board definition as [described below](#board-definitions).
 ## Advanced Usage
 
 The above commands assume that your MCU connects at the default baud rate
-of 250000 and the firmware is located at `~/klipper/out/klipper.bin`.  The
+of 250000 and the firmware is located at `out/klipper.bin`.  The
```

Comment:
This is explicitly documenting that the tool looks at an absolute path.  The tool does not look at `out/klipper.bin` so this change would be an error.  In general, it's probably best to not change the paths in the docs unless they are obviously incorrect.

Reply:
I have reverted this one - I used "replace all" without checking the script. Sry